src/controllers/notSelectedSignatory.controller.ts

```diff
@@ -1,13 +1,46 @@
+import { inject } from 'inversify'
 import { controller, httpGet } from 'inversify-express-utils'
 
 import BaseController from 'app/controllers/base.controller'
+import ViewApplicationStatusMapper from 'app/mappers/view-application-status/viewApplicationStatus.mapper'
+import DissolutionGetResponse from 'app/models/dto/dissolutionGetResponse'
+import Optional from 'app/models/optional'
+import DissolutionSession from 'app/models/session/dissolutionSession.model'
+import { ViewApplicationStatus } from 'app/models/view/viewApplicationStatus.model'
 import { NOT_SELECTED_SIGNATORY } from 'app/paths'
+import DissolutionService from 'app/services/dissolution/dissolution.service'
+import SessionService from 'app/services/session/session.service'
+
+interface ViewModel {
+  viewApplicationStatus: ViewApplicationStatus
+}
 
 @controller(NOT_SELECTED_SIGNATORY)
 export class NotSelectedSignatoryController extends BaseController {
 
+  public constructor(
+    @inject(SessionService) private session: SessionService,
+    @inject(DissolutionService) private dissolutionService: DissolutionService,
+    @inject(ViewApplicationStatusMapper) private viewApplicationStatusMapper: ViewApplicationStatusMapper
+  ) {
+    super()
+  }
+
   @httpGet('')
   public async get(): Promise<string> {
-    return super.render('not-selected-signatory')
+    const token: string = this.session.getAccessToken(this.httpContext.request)
+    const dissolutionSession: DissolutionSession = this.session.getDissolutionSession(this.httpContext.request)!
+
+    const dissolution: Optional<DissolutionGetResponse> = await this.dissolutionService.getDissolution(token, dissolutionSession)
+
+    return this.renderView(dissolution!)
+  }
+
+  private async renderView(dissolution: DissolutionGetResponse): Promise<string> {
+    const viewModel: ViewModel = {
```

Comment:
Same here as mentioned previously: define and then use, rather than combining both. This is purely a stylistic preference though so it's up to you.

Reply:
Done